utils/train_utils.py

```diff
@@ -250,6 +267,7 @@ def setup_environ_flags(rank):
     os.environ["TORCH_SHOW_CPP_STACKTRACES"] = str(1)
     os.environ["NCCL_ASYNC_ERROR_HANDLING"] = str(1)
     os.environ["TORCH_DISTRIBUTED_DEBUG"] = "DETAIL"
+    os.environ['PYTORCH_CUDA_ALLOC_CONF']='expandable_segments:True'
```

Comment:
expandable_segments is only available for the PyTorch Nightlies at present. TORCH_DISTRIBUTED_DEBUG=DETAIL will be too verbose for regular training. Will be better to document these for additional troubleshooting tips

Reply:
Sure, I will add them to the FAQ and just comment them here instead of removing to leave some hints for user.